### DISCUSSION TITLE
Can my config work ?

### DISCUSSION THREAD

**Discussion by corazon008** [OpenVoiceOS/ovos-buildroot#281] [Q&A]
Hello,
It's not my first project with raspberry pi but it is in voices assistants.
From what I can read from the website the supported hardwares are 
```
Raspberry Pi 3 (Including the Mark 1)
Raspberry Pi 4 - Mark 2 / Mark 2 Dev Kit / ReSpeaker x86-64
```
And the supported peripherals are
```
    ReSpeaker 2-Mic
    ReSpeaker 4-Mic Array
    ReSpeaker 4-Mic Linear Array
    ReSpeaker 6-Mic Circular Array
    RPI-Proto (Mark 1 soundcard)
    SJ-201 (Mark 2 LED/audio/microphone board)
    PS3-Eye (USB microphone)
    Google AIY-v1 / Chatterbox HAT
```
But I don't know if what I have is one of these.
I own a rpi 3 B+ (I don't know what is Mark 1) and I am willing to buy this little [mic](https://www.amazon.fr/SunFounder-Microphone-Raspberry-Recognition-Software/dp/B01KLRBHGM?__mk_fr_FR=%C3%85M%C3%85%C5%BD%C3%95%C3%91&crid=2D2CXPIDLD4K&dib=eyJ2IjoiMSJ9.Zrq0wwcNjl7P9fSaHhLp-1-Au7na2Rf8WxEU4Pw9Za4PYoTzWbO5MSC4JwgdfPNKp2W-oZUiz_MzzyBYMUpxkh3jrHHgQtsaeKONAPGZYZVK3H7Cj4UOqBVTDIh2DxdO86mtj7opZSWx5_aRZ3dC6NgQmDfqsZrG0f7gneh7n60NMuEhHt0lHk_1JoSmjlIvkqPkQkTyE3IYZ4kU-ukP6wETl6ik40tkFP7AoaJTCpiEt1VPlZIeJH1NOKXvEk_uGWmgiZgoYuNPLDAiv-c9SQ5RaVigyYcow3x0ovNWUfw.06G8SLFtgwqwvBT-mlZ7J4PjKF8IzNl5MK5g59pfTIk&dib_tag=se&keywords=rpi+microphone&nsdOptOutParam=true&qid=1732653753&sprefix=rpi+microphone%2Caps%2C117&sr=8-5) and to use the jack port as a speaker (obviously I have an external speaker).

Also, from the comments I read on Amazon, we need to be close to the mic in order to work properly, which is not realy suitable. That's why if you have an alternative cheaper than 25$ if possible.

Thank you

**Comment by mjlill** [comment]
The Pi3B+ works fine. I increased the swap to1024  as sometimes the little 100 isn't quite enough.  I have used a Logitech webcam as a mic, but prefer the Google AIY V1 kit. Either mic hears me from across the room.
